### ISSUE TITLE
Error log is not displayed

### ISSUE THREAD

**Issue by SergejGrin** [WWBN/AVideo#8945]
Error log is not displayed. The error file is created, but when I click on Error File in the side menu of the site, instead of the log, I get this window (see screenshot).

<img width="1320" alt="2024-04-02" src="https://github.com/WWBN/AVideo/assets/59236880/af842a48-52f8-42d2-a77a-a52820c69521">

Before this, a backup plugin was installed and running, which had a problem. (see [issues/8858(url=https://github.com/WWBN/AVideo/issues/8858)

**Comment by DanielnetoDotCom** [comment]
is is probably too big
 
try to delete the avideo.log or use command line to access and check it

**Comment by SergejGrin** [comment]
Thanks Daniel, this helped.
I had to delete the log file and cache folder FOUR times and even (for the purity of the experiment) reboot the virtual machine, only then the page for viewing the error log became available again. After the last update and installation of the backup plugin, the error log began to grow very quickly.

<img width="1340" alt="2024-04-02-1" src="https://github.com/WWBN/AVideo/assets/59236880/3aa0d709-71df-416a-8533-b09dee959bd6">

**Comment by DanielnetoDotCom** [comment]
but what is logging in there to make it grow?

**Comment by DanielnetoDotCom** [comment]
the error in the ru.php does not make sense

try to run this 

```
cd /var/www/html/AVideo/Encoder
sudo git fetch --all && sudo git reset --hard origin/master

```

**Comment by SergejGrin** [comment]
But my encoder is located on a separate server/machine
<img width="991" alt="2024-04-02 в 21 17 32" src="https://github.com/WWBN/AVideo/assets/59236880/e0983520-09ad-4099-ab64-7f02f0f25001">

**Comment by DanielnetoDotCom** [comment]
Sorry, I mean 

```
cd /var/www/html/AVideo/
sudo git fetch --all && sudo git reset --hard origin/master

```

**Comment by SergejGrin** [comment]
Thanks again Daniel,
I ran this and deleted the old log file. The error log page no longer fills with countless entries.
<img width="1339" alt="2024-04-02 в 22 07 12" src="https://github.com/WWBN/AVideo/assets/59236880/06f6fabf-d1f2-4482-aed8-91b54a25d421">

**Comment by SergejGrin** [comment]
The error log opens, but for some reason it gives errors again...
<img width="1553" alt="2024-04-03 в 14 37 56" src="https://github.com/WWBN/AVideo/assets/59236880/2490ea5c-5aa8-450b-bd81-eea8000bf04b">

**Comment by SergejGrin** [comment]
A similar situation is on my second site. I can’t understand what it is and where to look?
<img width="1553" alt=" 2024-04-03 в 14 56 40" src="https://github.com/WWBN/AVideo/assets/59236880/1134961e-eb24-4682-a5dd-52827146c893">

**Comment by DanielnetoDotCom** [comment]
Look like you have some useless crawlers searching your site, and it is consuming resources. 

I guess you should block spiders, bots, and crawlers

https://github.com/WWBN/AVideo/wiki/Blocking-Bots-on-Your-AVideo-Platform

**Comment by SergejGrin** [comment]
Indeed, very similar to bots. Thanks for the tip, I'll look in this direction.
